s3tests_boto3/functional/test_iam.py

```diff
@@ -1780,6 +1793,63 @@ def test_account_role_policy(iam_root):
     with pytest.raises(iam_root.exceptions.NoSuchEntityException):
         iam_root.delete_role_policy(RoleName=role_name, PolicyName=policy_name)
 
+@pytest.mark.role_policy
+@pytest.mark.iam_account
+def test_account_role_policy_managed(iam_root):
+    path = get_iam_path_prefix()
+    name = make_iam_name('name')
+    policy1 = 'arn:aws:iam::aws:policy/AmazonS3FullAccess'
```

Comment:
aren't these managed policies supposed to exist somewhere?

Reply:
there are a ton of 'aws managed policies' listed in https://docs.aws.amazon.com/aws-managed-policy/latest/reference/policy-list.html that exist in aws

i chose a small number of those to hard-code into rgw in https://github.com/ceph/ceph/pull/54333/commits/75d295221b203620a1d169c85ae46ff5857dda8e

basically just the FullAccess and ReadOnlyAccess policies for S3, SNS, STS, IAM

Reply:
ok, how about `Customer Managed Policies`, do we have any plan to support that?

Reply:
> ok, how about `Customer Managed Policies`, do we have any plan to support that?

i hadn't planned to here. we'd need an efficient way to store them somewhere in rados and make them available to each request that uses them

i added these aws-managed ones because iam users/roles start with no policy, and the FullAccess and ReadOnlyAccess ones are a really convenient way to add blanket identity policies. they're also "free" to implement since we don't have to load the policy from rados